framework/db/oci/Schema.php

```diff
@@ -219,7 +215,15 @@ protected function createColumn($column)
             if (stripos($column['DATA_DEFAULT'], 'timestamp') !== false) {
                 $c->defaultValue = null;
             } else {
-                $c->defaultValue = $c->phpTypecast($column['DATA_DEFAULT']);
+                $defaultValue = $column['DATA_DEFAULT'];
+                if ($defaultValue !== null) {
+                    if (($len=strlen($defaultValue)) > 2 && $defaultValue[0] === "'" && $defaultValue[$len-1] === "'") {
+                        $defaultValue = substr($column['DATA_DEFAULT'], 1, -1);
```

Comment:
We only check if the value is surrounded with single quotes, which can't be multibyte. I don't think it matters if the rest of the value is.